docs/docsite/rst/dev_guide/testing_units_modules.rst

```diff
@@ -341,33 +352,32 @@ Running the main function
 If you do want to run the actual main function of a module you must import the module, set
 the arguments as above, set up the appropriate exit exception and then run the module::
 
-    # This test is based around pytest's features for individual test functions
-    import pytest
-    import ansible.modules.module.group.my_modulle as my_module
+    import ansible.modules.topic.subtopic.my_module as my_module
+    from units.modules.utils import TestAnsibleModule, set_module_args
 
-    def test_main_function(monkeypatch):
```

Comment:
Definitely we shouldn't make this change if the discussion in the Ansible Testing WG is correct.